src/Library/demos/Calendar/main.js

```diff
@@ -0,0 +1,27 @@
+const calendar = workbench.builder.get_object("calendar");
+
+function handleChange() {
+  let year = calendar.year;
+  let month = calendar.month;
+  let day = calendar.day;
+  let monthString = month.toString().padStart(2, "0");
+  let dayString = day.toString().padStart(2, "0");
+
+  console.log(
+    `Selected Date (YYYY-MM-DD): ${year}-${monthString}-${dayString}`,
+  );
+}
+
+calendar.connect("day-selected", handleChange);
+calendar.connect("next-month", () => {
+  console.log("Switched to Next Month");
+});
+calendar.connect("next-year", () => {
+  console.log("Switched to Next Year");
+});
+calendar.connect("prev-month", () => {
+  console.log("Switched to Previous Month");
+});
+calendar.connect("prev-year", () => {
+  console.log("Switched to Previous Year");
+});
```

Comment:
I can't really think of use case for these signals

I'd remove them - @andyholmes wdyt?

Reply:
maybe we can connect to property changes instead

```
calendar.connect('notify::month', () => {
  console.log(calendar.month);
});
```

Reply:
but it would return an integer , i don't think that would be relevant to log.

Reply:
You're right. What about we log the day/month/year in the user locale?